package.json

```diff
@@ -22,7 +22,7 @@
     "husky": "^4.2.3",
     "lint-staged": "^10.1.3",
     "prettier": "1.19.1",
-    "prettier-plugin-solidity": "1.0.0-alpha.46",
+    "prettier-plugin-solidity": "1.0.0-alpha.49",
```

Comment:
I verified at https://semver.npmjs.com/ that this semver matches the version we want.